src/Backend/Modules/Pages/Engine/Model.php

```diff
@@ -918,8 +918,12 @@ public static function getTypes(): array
         ];
     }
 
-    public static function getURL(string $url, int $id = null, int $parentId = 0, bool $isAction = false): string
+    public static function getURL(string $url, int $id = null, int $parentId = null, bool $isAction = false): string
     {
+        if ($parentId === null) {
+            $parentId = 0;
+        }
+
         $parentIds = [$parentId];
```

Comment:
you can remove the code above if you change this to
`$parentIds = [$parentId ?? 0];`